#! /bin/sh /usr/share/dpatch/dpatch-run
## bashism.dpatch by Samuel Mimram <smimram@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove bashisms.

@DPATCH@
diff -urNad menhir-20060615.dfsg~/Makefile menhir-20060615.dfsg/Makefile
--- menhir-20060615.dfsg~/Makefile	2006-06-15 07:26:06.000000000 +0000
+++ menhir-20060615.dfsg/Makefile	2006-07-11 12:24:59.000000000 +0000
@@ -15,21 +15,21 @@
 # Locating the ocaml compilers.
 # If ocamlfind is available, then it is used for that purpose.
 
-CAMLC           := $(shell if which ocamlfind &>/dev/null ; \
+CAMLC           := $(shell if which ocamlfind 2>&1 >/dev/null ; \
                        then echo ocamlfind ocamlc ; \
-		       elif which ocamlc.opt &>/dev/null ; \
+		       elif which ocamlc.opt 2>&1 >/dev/null ; \
                        then echo ocamlc.opt ; \
 		       else echo ocamlc ; fi)
 
-CAMLOPT         := $(shell if which ocamlfind &>/dev/null ; \
+CAMLOPT         := $(shell if which ocamlfind 2>&1 >/dev/null ; \
                        then echo ocamlfind ocamlopt ; \
-		       elif which ocamlopt.opt &>/dev/null ; \
+		       elif which ocamlopt.opt 2>&1 >/dev/null ; \
                        then echo ocamlopt.opt ; \
 		       else echo ocamlopt ; fi)
 
-CAMLDEP         := $(shell if which ocamlfind &>/dev/null ; \
+CAMLDEP         := $(shell if which ocamlfind 2>&1 >/dev/null ; \
                        then echo ocamlfind ocamldep ; \
-		       elif which ocamldep.opt &>/dev/null ; \
+		       elif which ocamldep.opt 2>&1 >/dev/null ; \
                        then echo ocamldep.opt ; \
 		       else echo ocamldep ; fi)
 
